SynthSeg/training_with_tfrecords.py

```diff
@@ -22,6 +25,10 @@ class NullStrategy:
     def scope():
         return nullcontext()
 
+def set_seed(seed = 42):
```

Comment:
Maybe we could expose the `seed` argument in the training options.

Reply:
great suggestion, done!